packages/next-swc/crates/next-dev-tests/tests-manifest.js

```diff
@@ -23,247 +26,186 @@ const enabledTests = [
-  'test/integration/render-error-on-top-level-error/with-get-initial-props/test/index.test.js',
-  'test/integration/render-error-on-top-level-error/without-get-initial-props/test/index.test.js',
-  'test/integration/required-server-files-ssr-404/test/index.test.js',
-  'test/integration/revalidate-as-path/test/index.test.js',
-  'test/integration/rewrites-destination-query-array/test/index.test.js',
-  'test/integration/root-optional-revalidate/test/index.test.js',
-  'test/integration/route-indexes/test/index.test.js',
-  'test/integration/route-load-cancel-css/test/index.test.js',
-  'test/integration/route-load-cancel/test/index.test.js',
   'test/integration/router-hash-navigation/test/index.test.js',
-  'test/integration/router-is-ready-app-gip/test/index.test.js',
-  'test/integration/router-is-ready/test/index.test.js',
   'test/integration/router-prefetch/test/index.test.js',
-  'test/integration/scss/test/group-2.test.js',
+  'test/integration/router-rerender/test/index.test.js',
+  'test/integration/scss/test/dev-css-handling.test.js',
   'test/integration/src-dir-support-double-dir/test/index.test.js',
-  'test/integration/src-dir-support/test/index.test.js',
-  'test/integration/ssg-data-404/test/index.test.js',
   'test/integration/ssg-dynamic-routes-404-page/test/index.test.js',
   'test/integration/static-404/test/index.test.js',
   'test/integration/static-page-name/test/index.test.js',
-  'test/integration/styled-jsx-plugin/test/index.test.js',
+  'test/integration/trailing-slashes-href-resolving/test/index.test.js',
   'test/integration/tsconfig-verifier/test/index.test.js',
-  'test/integration/telemetry/test/page-features.test.js',
-  'test/integration/turbopack-unsupported-log/index.test.ts',
-  'test/integration/turbotrace-with-webpack-worker/test/index.test.js',
-  'test/integration/typeof-window-replace/test/index.test.js',
   'test/integration/typescript-baseurl/test/index.test.js',
-  'test/integration/typescript-custom-tsconfig/test/index.test.js',
+  'test/integration/typescript-external-dir/project/test/index.test.js',
   'test/integration/typescript-filtered-files/test/index.test.js',
-  'test/integration/typescript-ignore-errors/test/index.test.js',
-  'test/integration/webpack-config-extensionalias/test/index.test.js',
+  'test/integration/typescript-hmr/test/index.test.js',
   'test/integration/webpack-config-mainjs/test/index.test.js',
+  'test/integration/with-electron/test/index.test.js',
+  'test/production/ci-missing-typescript-deps/index.test.ts',
+  'test/production/enoent-during-require/index.test.ts',
+  'test/production/eslint-plugin-deps/index.test.ts',
+  'test/production/fallback-export-error/index.test.ts',
+  'test/production/jest/new-link-behavior.test.ts',
+  'test/production/jest/transpile-packages.test.ts',
+  'test/production/postcss-plugin-config-as-string/index.test.ts',
+  'test/production/supports-module-resolution-nodenext/supports-moduleresolution-nodenext.test.ts',
```

Comment:
Why are these being added if they are production tests?

Reply:
These work with turbopack. It's using the experimental support for build, but green and runs turbopack